### ISSUE TITLE
Sound files visible despite they set invisible in Spoke

### ISSUE THREAD

**Issue by navratiljudit** [mozilla/Spoke#1227]
Since the new audio feature, old Hubs rooms show the audio files (as black rectangle with note) despite they are set invisible in Spoke. how can I make them invisible but audible again?

**Comment by minim-sheba** [comment]
Yes, I'm having this problem too - my 'invisible' audio is correctly hidden in Spoke, but visible when I publish to Hubs. Someone in another thread mentioned setting scale to 0, which seems to hide it OK, but it would be great if this could be fixed to work as expected. Thanks.
